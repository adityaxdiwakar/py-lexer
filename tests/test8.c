void greet() {
    printf("Hello, World!\n");
}

int main() {
    greet();
    
    return 0;
}