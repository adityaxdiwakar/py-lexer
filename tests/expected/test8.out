VOID_WORD
IDENT(greet)
LPAREN
RPAREN
LBRACK
IDENT(printf)
LPAREN
STR("Hello, World!\n")
RPAREN
SEMI
RBRACK
INT
MAIN
LPAREN
RPAREN
LBRACK
IDENT(greet)
LPAREN
RPAREN
SEMI
RETURN
INT(0)
SEMI
RBRACK